### ISSUE TITLE
AndroidTV: Элементы управления плеером автоматически не скрываются после выключения паузы 

### ISSUE THREAD

**Issue by Ofigelov** [anilibria/anilibria-app#39]
Если поставил на паузу а потом включаешь воспроизведение, то в 95% случаев элементы управления не скрываются, и надо дополнительно на пульте нажать кнопку "назад". Эта мелочь очень сильно раздражает, особенно когда за сеанс просмотра ставишь паузу несколько раз.

**Comment by RadiationX** [comment]
Да, есть такое, тоже немного раздражает это. На момент разработки было не просто это исправить, поэтому оставил как есть.
Когда буду заниматься доработками, обязательно гляну этот момент, постараюсь исправить.
По срокам не сориентирую.

**Comment by RadiationX** [comment]
Будет исправлено в ближайшем релизе, скорее всего до нового года.
Скорее всего опубликую тестовую версию, будет возможность проверить её?
